code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -404,7 +413,7 @@ DEFINE_BITFIELD(turret_flags, list(
 				cover = new /obj/machinery/porta_turret_cover(loc)	//if the turret has no cover and is anchored, give it a cover
 				cover.parent_turret = src	//assign the cover its parent_turret, which would be this (src)
 
-	if(!on || (machine_stat & (NOPOWER|BROKEN)) || manual_control)
+	if(!on || (machine_stat & (NOPOWER|BROKEN)) || manual_control || (!stored_gun && uses_stored))
```

Comment:
This is getting confusing, split this into multiple if statements.

Reply:
it is now three